src/common/util.c

```diff
@@ -229,3 +230,30 @@ char *oscap_expand_ipv6(const char *input)
 
 	return ret;
 }
+
+char *oscap_path_join(const char *path1, const char *path2)
+{
+	if (path1 == NULL) {
+		return oscap_strdup(path2);
+	}
+	if (path2 == NULL) {
+		return oscap_strdup(path1);
+	}
+	size_t path1_len = strlen(path1);
+	size_t path2_len = strlen(path2);
```

Comment:
const

Reply:
I think it is changed below

Reply:
Yes, both path1_len and path2_len are changed a few lines below.